blockbase/inc/customizer/wp-customize-colors-preview.js

```diff
@@ -1,4 +1,8 @@
-var enabledDuotone = duotoneVars[ 'duotoneControl' ] === '1' ? true : false;
+var enabledDuotone = false;
+
+if ( window.duotoneVars ) {
```

Comment:
does this need to be window.duotoneVars? Couldn't it just be `duotoneVars`?

Reply:
it didn't work that way!

> Uncaught ReferenceError: duotoneVars is not defined

Reply:
I think you need to check the type of duotoneVars:

```
if ( typeof duotoneVars !== 'undefined' ) {
```

but I suppose your way works too